tools/convert_sequential_to_hf.py

```diff
@@ -157,7 +157,16 @@ def convert(input_checkpoint_path, loaded_config, output_checkpoint_path):
 
     hf_model = GPTNeoXForCausalLM(
         hf_config
-    ).half()  # nice-to-have: lazy init weights somehow?
+    )
+
+    # save model in FP16 if Deepspeed fp16 was used in config, else 32 bit
+    fp16 = get_key(loaded_config, "fp16")
```

Comment:
Should there be logic here to handle bf16 explicitly? Even if it’s not mechanically necessary it’s probably going to be clearer to future users and maintainers.

Reply:
Good call, will fix!